### PR TITLE
A couple of Typos. Html syntax fixes. .gitignore.

### PR DESCRIPTION
**Pull request by mikelittle**

### CODE REVIEW

File: code-of-conduct.html [hunk 1]
```diff
@@ -47,28 +47,27 @@
...
     <li>Communicating in a ‘tone’ you don’t find congenial</li>
     <li>Criticizing racist, sexist, cissexist, or otherwise oppressive behavior or assumptions </li>
   </ul>
-</p>
 
-<h2>Reporting</h2>
+<h3>Reporting</h3>
 
-<p>If you are being harassed by a member of Web Matters, notice that someone else is being harassed, or have any other concerns, please contact us. If the person who is harassing you is on the team, they will recuse themselves from handling your incident. We will respond as promptly as we can.</p>
+<p>If you are being harassed by a member of Web Matters, notice that someone else is being harassed, or have any other concerns, please contact us. If the person who is harassing you is on the team, they will excuse themselves from handling your incident. We will respond as promptly as we can.</p>
```
Comment (rightsaidjames) [inline code-of-conduct.html:62]: Why `excuse` rather than `recuse`?

Comment (rightsaidjames) [reply code-of-conduct.html:62]: Given that this wording comes from someone else's CoC (Geek Feminism, I think Orde said) and either word has broadly the same meaning, perhaps we could take out this change for now in order to get the rest of the fixes merged?

Comment (mikelittle) [reply code-of-conduct.html:62]: The dictionary definition of recuse I looked up described it as a chiefly North American usage. The explanation of the word uses the phrase "excuse oneself". For a UK-based organisation it seemed more appropriate.

Comment (rightsaidjames) [reply code-of-conduct.html:62]: Fair enough! I must have read so much North American legalese over the years that 'recuse' was the word that looked 'right' to me, even though both have the same meaning.

**Review by rightsaidjames — APPROVED** [review]

**Pull request merged**